plugin.rb

```diff
@@ -108,12 +108,21 @@ def ensure_models
   end
 
   post '/IpamDriver.ReleaseAddress' do
-    Addresses::Release.run!(
-      pool_id: params['PoolID'],
-      address: params['Address']
-    )
-
-    json({})
+    begin
+      Addresses::Release.run!(
+        pool_id: params['PoolID'],
+        address: params['Address']
+      )
+
+      json({})
+    rescue Mutations::ValidationException => exc
+      if exc.errors['address'] && exc.errors['address'].symbolic == :zombie
+        status 202
```

Comment:
I guess this is one of the cases where none of the existing HTTP status codes really match well. :/ 409 kinda fits as in "Yes, the address should be released but cannot as it's in conflicting state (still responds to ping)".

I'm ok using 409. @jakolehm agree?